packages/kbn-securitysolution-exception-list-components/README.md

```diff
@@ -0,0 +1,27 @@
+# @kbn/securitysolution-exception-list-components
+
+This is where the building UI components of the Exception-List live
+Most of the components here are imported from `x-pack/plugins/security_solutions/public/detection_engine` 
+
+# Aim 
+
+TODO
+
+# Pattern used
+
+```
+component
+    index.tsx
+    index.styles.ts <-- to hold styles if the component has many custom styles
+    useComponent.ts <-- for logic if the Presentational Component has logic
```

Comment:
should this be camel case to follow how the other file naming works? `use_component.ts`

Reply:
yes good point